dapp_methods_calls.py

```diff
@@ -0,0 +1,68 @@
+from pyteal import *
+
+
+def methods_calls(methods: dict, call_idx: int):
```

Comment:
This could use annotations for the dict `methods: dict[str, Expr]` so it's extra clear for users (assuming python 3.9)